java/Ice/greeter/server/src/main/java/com/zeroc/greeter/Chatbot.java

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) ZeroC, Inc.
+
+package com.zeroc.greeter;
+
+import com.zeroc.greeter.VisitorCenter.Greeter;
+import com.zeroc.Ice.Current;
+
+/**
+ * Chatbot is an Ice servant that implements Slice interface Greeter.
+ */
+class Chatbot implements Greeter
```

Comment:
Also feels public to me.
```suggestion
public class Chatbot implements Greeter
```

Reply:
We use internal for C#

```
internal class Chatbot : GreeterDisp_
```